plugin/chatgpt.vim

```diff
@@ -84,9 +89,10 @@ function! ChatGPT(prompt) abort
 
 def chat_gpt(prompt):
   max_tokens = int(vim.eval('g:chat_gpt_max_tokens'))
-  model= str(vim.eval('g:chat_gpt_model'))
+  model = str(vim.eval('g:chat_gpt_model'))
+  lang = str(vim.eval('g:chat_gpt_lang'))
   temperature = float(vim.eval('g:chat_gpt_temperature'))
-  systemCtx = {"role": "system", "content": "You are a helpful expert programmer we are working together to solve complex coding challenges, and I need your help. Please make sure to wrap all code blocks in ``` annotate the programming language you are using."}
+  systemCtx = {"role": "system", "content": f"You are a helpful expert programmer we are working together to solve complex coding challenges, and I need your help. Please make sure to wrap all code blocks in ``` annotate the programming language you are using. And respond in {lang}"}
```

Comment:
can you make this is conditional so that it does not append it to the context if the `lang` is empty?

Reply:
my mistake. I've udpate a new push, plz check.